lib/ansible/modules/cloud/amazon/lambda.py

```diff
@@ -48,7 +47,7 @@
     description:
       - The Amazon Resource Name (ARN) of the IAM role that Lambda assumes when it executes your function to access any other Amazon Web Services (AWS)
         resources. You may use the bare ARN if the role belongs to the same AWS account.
-    default: null
+    required: true
```

Comment:
I've now moved to having everything only when state==present and updating the docs appropriately.